docker-compose.yml

```diff
@@ -16,6 +16,7 @@ services:
     ports:
       - 4000:4000
       - 4010:4010
+      - 9229:9229
```

Comment:
This seems to be default Node debugger port so I mapped it. we can remove 4010 as it is redundant.